### DISCUSSION TITLE
error: Invalid character in header content ["Content-disposition"]

### DISCUSSION THREAD

**Discussion by 0v3rd0z3d** [requarks/wiki#4929] [Error / Bug Report]
![image](https://user-images.githubusercontent.com/65501113/150765398-db697628-c45c-495b-a3d0-d0cf84c02a73.png)
![image](https://user-images.githubusercontent.com/65501113/150765497-7eff5aff-7eab-4d73-b260-56ba28418f21.png)
Acquired an"internal server error" while trying to open assets. Names are in Russian characters.
Not having this error with filenames in english characters.
Versions of wikijs with this issue: 2.5.268, 2.5.272
Not having this error in version: 2.5.201
Thank you in advance!

**Comment by kain90** [comment]
Yes, I have the same problem with some files which name has special characters. Names are in Spanish characters (áéíóú)

The only solution that works is downgrade the WikiJS version that you mention (2.5.201).

Thank you!
